src/submit-modal/submit-modal.module.css

```diff
@@ -0,0 +1,23 @@
+.submitModalSummaryWrapper {
```

Comment:
`color: grey900` needs to be set, as the `ModalContent` doesn't seem to set it itself. (All content inside the modal should be `color: grey900` unless otherwise specified, so not sure if there's a more generic place to define this? Right now all is #000)